core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -156,6 +155,9 @@ class TransactionMarkerChannelManager(config: KafkaConfig,
   }
 
   // visible for testing
+  private[transaction] def queueForUnknownBroker = markersQueueForUnknownBroker
```

Comment:
Hiding the field doesn't seem to have much value if we expose it directly through an accessor anyway and it's annoying to need two names for the same thing. If it only needs to be accessed from a test case, I wouldn't worry too much about violating encapsulation because we can just change the test case if the implementation changes. That said, it's just a nit, so feel free to ignore.